crates/ty_ide/src/completion.rs

```diff
@@ -5812,13 +5812,7 @@ from .imp<CURSOR>
         let builder = completion_test_builder("deprecated<CURSOR>")
             .auto_import()
             .module_names();
-        assert_snapshot!(builder.build().snapshot(), @r"
-        Deprecated :: importlib.metadata
-        DeprecatedList :: importlib.metadata
-        DeprecatedNonAbstract :: importlib.metadata
-        DeprecatedTuple :: importlib.metadata
```

Comment:
What change is causing these suggestions to go away? They don't seem to be defined in an underscore-prefixed module. They are defined in a `sys.version_info` conditional -- do we also add support for that in this PR? Or change the Python version we run these tests with?